Bottom: 0f709b58fab145293e131b521ae73121056f1cfe
Top:    bb02bd99c67f1a2740ffbf00c8d1c4bfffbceb37
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-06 15:54:08 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
